cockroachdb/manifests-cert-manager/db-init/cockroach-db-init.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: &app cockroach-init
+  labels:
+    version: v1
+    app: *app
+spec:
+  template:
+    spec:
+      containers:
+        - name: db-init
+          image: cockroachdb
+          imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              cpu: 0
+              memory: 128Mi
+            limits:
+              cpu: 1
+              memory: 512Mi
+          command:
+            - "/bin/bash"
+            - "/opt/scripts/user-schema-bootstrap.sh"
+          env:
+            - name: COCKROACH_HOST
+              valueFrom:
+                configMapKeyRef:
+                  name: cockroach
+                  key: cockroach.host
+            - name: COCKROACH_PORT
+              valueFrom:
+                configMapKeyRef:
+                  name: cockroach
+                  key: cockroach.port
+            - name: COCKROACH_CERTS_DIR
+              value: "/cockroach-certs/"
+            # This variables should be replaced by patches
+            - name: DB_NAME
+              value: "default"
```

Comment:
why do we mix some values being set from configMap and some being patched in, what are the different use cases for each approach?

Reply:
It's db init job- it's used as a base to create new jobs for each new db/user. 
Some of the env will be the same for each of jobs (set from CM), while some needs to be customized (I almost spelled it through `k`, lol), and those are patched.

Would you suggest documenting this better?

Reply:
Speaking of documentation, do we reference these examples in READMEs?

Reply:
I plan to rewrite most of the documentation, so that it does fit the current formula from shared-kustomize-bases. Hence, I'll do it it [another ticket](https://linear.app/utilitywarehouse/issue/DENA-600/update-cockroachdb-documentation)